shared-services/README.md

```diff
@@ -96,6 +101,10 @@ future changes by simply running `terraform apply
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
+| assessment\_findings\_bucket\_name | The name of the assessment findings S3 bucket. | `string` | n/a | yes |
+| assessment\_findings\_bucket\_object\_name\_pattern | The name pattern specifying which objects are allowed to be written to the assessment findings data S3 bucket. | `string` | `"*-data.json"` | no |
```

Comment:
Now into nit-picky things but should we do

```suggestion
| assessment\_findings\_bucket\_object\_key\_pattern | The key pattern specifying which objects are allowed to be written to the assessment findings data S3 bucket. | `string` | `"*-data.json"` | no |
```

to align with AWS terminology?